llama_index/llms/loading.py

```diff
@@ -0,0 +1,41 @@
+from llama_index.llms.base import LLM
+from llama_index.llms.custom import CustomLLM
+from llama_index.llms.huggingface import HuggingFaceLLM
+from llama_index.llms.langchain import LangChainLLM
+from llama_index.llms.llama_cpp import LlamaCPP
+from llama_index.llms.mock import MockLLM
+from llama_index.llms.openai import OpenAI
+from llama_index.llms.palm import PaLM
+from llama_index.llms.predibase import PredibaseLLM
+from llama_index.llms.replicate import Replicate
+from llama_index.llms.xinference import Xinference
+
+
+def load_llm(data: dict) -> LLM:
+    """Load LLM by name."""
+    llm_name = data.get("class_name", None)
```

Comment:
A general comment about using the class_name as the identifying attribute in the serialized representation is that now if you rename the class, it breaks compatibility with previously serialized representations.

An alternative is to keep an explicit mapping of type identifier to class: e.g.
```
mapping = {
  'openai': OpenAI,
   ...
}

Reply:
I think it's not a big concern yet, so okay with the current approach of just using class name

Reply:
Done!